data/json/items/tool/workshop.json

```
@@ -1615,5 +1672,17 @@
     "material": [ "steel" ],
     "symbol": "&",
     "color": "light_gray"
+  },
+  {
+    "id": "welding_blanket",
+    "type": "TOOL",
+    "name": { "str": "welding blanket" },
+    "description": "A non-inflammable mat used to protect surfaces from welding splatter or to insulate a weld when repairing tempered metal.  This one is made with asbestos.",
```

```suggestion
    "description": "A non-flammable mat used to protect surfaces from welding splatter or to insulate a weld when repairing tempered metal.  This one is made with asbestos.",
```
Current wording is correct as well, but people often think that "inflammable" means "not flammable" (as I personally did as well until now, though English isn't my native language), so this could be made less ambiguous.